skimage/util/tests/test_arraypad.py

```diff
@@ -2,19 +2,12 @@
 
 import numpy as np
```

Comment:
Do we need to keep these tests for `np.pad`?

Reply:
Yeah, I don't see why we need to keep these. If there are deficiencies in NumPy's `pad` tests we should just improve them there.

Reply:
I have now removed this file.